src/Pagination/index.jsx

```diff
@@ -158,6 +158,7 @@ class Pagination extends React.Component {
         <Button
           className="page-link"
           aria-label={ariaLabel}
+          variant={active ? 'primary' : 'tertiary'}
```

Comment:
I think this is the right approach here, though I think there are some custom existing `Pagination` styles interfering with the expected button styles still. For example: 

<img width="539" alt="image" src="https://user-images.githubusercontent.com/2828721/208110363-e1c85bd3-2c37-4801-bc12-149177c4dd43.png">

The text color of the non-active "tertiary" button variants seem to be getting a blue color instead of the standard "tertiary" button variant styles:

<img width="604" alt="image" src="https://user-images.githubusercontent.com/2828721/208110658-f4af90da-8b06-4492-b55a-bb7f4e02b235.png">